src/io.rs

```diff
@@ -0,0 +1,198 @@
+use crate::units::preamble::*;
```

Comment:
The order of these imports look a bit backwards to me; I usually put (std, external crates, current workspace), per https://github.com/rust-dev-tools/fmt-rfcs/issues/131. But to my knowledge, there's no `cargo fmt` automation for this, only some IDE support.

Just FYI. Feel free to leave them this way; just aim for consistency in the rest of the crate then

Reply:
I actually do prefer that way around. I was distracted by this feeling very `prelude` like (that's what I meant to call it), so wanting to put it first.